src/components/ProjectView/ProjectImages.vue

```diff
@@ -16,10 +15,7 @@
 </script>
 
 <template>
-    <v-card>
         <!-- These divs are just for demonstration purposes, feel free to remove them when implemented -->
-        <div>{{ selectedProject }}</div>
-        <image-list :data="MockData" :selectedImages="selectedImages" @update-selected="updateSelectedImages"/>
-        <ImageCarousel></ImageCarousel>
-    </v-card>
+    <div>{{ selectedProject }}</div>
+    <image-list :data="MockData" :selectedImages="selectedImages" @update-selected="updateSelectedImages"/>
 </template>
```

Comment:
Missing final newline - VSCode will do this for you! https://stackoverflow.com/a/44704969